test/e2e/install_test.go

```diff
@@ -0,0 +1,61 @@
+package e2e
```

Comment:
```
package e2e_test
```
??

Reply:
I think package e2e is fine - there's only tests and test tooling here...no?

Reply:
Yeah that's right, I feel like I usually just see that convention when we have `actualpkg` and then when we test it `actualpkg_test`